base/logging_context.go

```diff
@@ -267,11 +267,14 @@ type EffectiveUserPair struct {
 type UserIDDomain string
 
 const (
-	UserDomainSyncGateway UserIDDomain = "sgw"
-	UserDomainCBServer    UserIDDomain = "cbs"
-	UserDomainBuiltin                  = "builtin" // internal users (e.g. SG bootstrap user)
+	UserDomainSyncGateway      UserIDDomain = "sgw"
+	UserDomainCBServer         UserIDDomain = "cbs"
+	UserDomainSyncGatewayAdmin UserIDDomain = "sgw_admin" // domain for SGW admin API when admin auth disabled
```

Comment:
I think I'd probably add `sgw_admin_noauth` here, but I don't feel strongly about this.